content/en/infrastructure/process/_index.md

```diff
@@ -232,6 +232,48 @@ I/O and open files stats can be collected by the Datadog system-probe, which run
    **Note**: If the `systemctl` command is not available on your system, run the following command instead: `sudo service datadog-agent restart`
 
 
+### Optimize footprint for process collection
+By default, the Datadog Agent has a separate Process Agent for container and process collection. You can consolidate container and process collection to the core Agent if you're running a Linux environment.
+
```

Comment:
Perhaps this isn't an issue considering this is an installation page, but should we specify the min versions of the agent and operator?

Reply:
I personally don't think it needs it since it's an install. If you'd like to add it, I suggest a separate PR since the current state already has docs approval.

Reply:
Ok sounds good. We can move forward then